osu.Game.Rulesets.Osu/Difficulty/Skills/AimWithoutSliders.cs

```diff
@@ -0,0 +1,18 @@
+// Copyright (c) ppy Pty Ltd <contact@ppy.sh>. Licensed under the MIT Licence.
+// See the LICENCE file in the repository root for full licence text.
+
+using osu.Game.Rulesets.Mods;
+
+namespace osu.Game.Rulesets.Osu.Difficulty.Skills
+{
+    /// <summary>
+    /// Represents the Aim skill that does not include sliders length in it's calculation
+    /// </summary>
+    public class AimWithoutSliders : Aim
```

Comment:
And this is... kinda weird? Why bother?

Why not expose `WithSliders` as public readonly on the skill, and then find it where you need it based on that flag alone? As in

```csharp
skills.OfType<Aim>().SingleOrDefault(aim => aim.WithSliders)
```

or something.

Reply:
It's mostly for the tools, to be able to distinguish between two aim skills

Reply:
Is that to say they can't distinguish by based on a property for some reason?

Reply:
Right now to display the skill name it just takes the type name. I could add an exception for aim, but I feel like it's an ugly way cos for example if someone works on aim revamping they'll need to remove it, or if someone add another multipass skill it'll need another exception and so on, and that'd need to happen not only in tools but in diffcalc itself. Creating a separate class seems like a more elegant solution to me

Reply:
Hmm. In general I'd expect that to be handled by a `Name` property or similar, but I suppose this is fine for now.